resources/views/filament/hooks/topbar.blade.php

```diff
@@ -14,7 +14,7 @@
         </x-slot>
 
         @if($bookmarks->isEmpty())
-            <x-filament::dropdown.header icon="tabler-bookmark-off">
+            <x-filament::dropdown.header :icon="config('zeus-delia.dropdown.empty_icon')">
```

Comment:
for older instlation the config key: `empty_icon` wont be available, I'll added to the doc and the relase notes, but its good to default to something like: `config('zeus-delia.dropdown.empty_icon','heroicon-m-bookmark-slash')`